sonar/modules/documents/views.py

```diff
@@ -43,33 +45,50 @@
 
 
 @blueprint.url_defaults
-def add_ir(endpoint, values):
-    """Add default ir parameter."""
-    values.setdefault('ir', 'sonar')
+def default_view_code(endpoint, values):
+    """Add default view code."""
+    values.setdefault('view',
+                      current_app.config.get('SONAR_APP_DEFAULT_ORGANISATION'))
 
 
 @blueprint.url_value_preprocessor
-def pull_ir(endpoint, values):
-    """Add ir parameter to global variables."""
-    g.ir = values.pop('ir')
+def store_organisation(endpoint, values):
+    """Add organisation record to global variables."""
+    view = values.pop('view',
+                      current_app.config.get('SONAR_APP_DEFAULT_ORGANISATION'))
+
+    if view != current_app.config.get('SONAR_APP_DEFAULT_ORGANISATION'):
+        organisation = OrganisationRecord.get_record_by_pid(view)
+
+        if not organisation or not organisation['isShared']:
+            raise Exception('Organisation\'s view is not accessible')
+
+        g.organisation = organisation.dumps()
 
 
 @blueprint.route('/')
 def index():
-    """IR (and SONAR) home view."""
+    """Homepage."""
     return render_template('sonar/frontpage.html')
 
 
-@blueprint.route('/search/<resource_type>')
-def search(resource_type=None):
-    """IR search results."""
+@blueprint.route('/search/documents')
+def search():
+    """Search results page."""
     return render_template('sonar/search.html')
 
 
-def detail(pid, record, template=None, **kwargs):
-    """Search details."""
-    g.ir = kwargs.get('ir')
-    return render_template('documents/record.html', pid=pid, record=record)
+@blueprint.route('/documents/<pid_value>')
```

Comment:
Seems a conflict with https://github.com/rero/sonar/blob/5af14d6daab1edb806264e8bae8708b58d003d00/sonar/config.py#L246

Reply:
I expressly removed it in this commit: https://github.com/rero/sonar/pull/234/files#diff-8cdd8777bd2125b8f78dfb443c855eedL246-L250.